src/_gettsim_tests/test_data/renten_alter.csv

```diff
@@ -0,0 +1,1369 @@
+jahr,p_id,hh_id,tu_id,geburtsjahr,geburtsmonat,weiblich,alter,m_arbeitsunfähig,m_krank_ab_16_bis_24,m_mutterschutz,m_arbeitslos,m_ausbild_suche,m_schul_ausbild,m_alg1_übergang,m_geringf_beschäft,y_pflichtbeitr_ab_40,m_freiw_beitrag,m_ersatzzeit,m_kind_berücks_zeit,m_pfleg_berücks_zeit,m_pflichtbeitrag,_ges_rente_altersgrenze_abschlagsfrei,Comment,,,Quelle,Calculation of age,yob,mob,regsray,regsram,ltsray,ltsram,wosray,wosram
+2010,1,1,1,1930,1,TRUE,80,0,0,0,0,0,0,0,0,10,0,0,0,0,180,60,Altersrente für Frauen,,,Table taken from a paper which calculates the retirement age for each cohort (variables with y indicate years and variables with m indicate months),(variable_y)+(variable_m)*(1/12),1900,1,65,0,63,0,60,0
```

Comment:
I would replace `a paper` by the names of the authors. Otherwise, the PR looks good and almost ready to merge!

Reply:
@TeBackh Do you have the names of the author?
Another open question was whether the conversion of the test cases is comprehensible enough, since I made them by hand, or whether we need a function, for example. What do you think @ChristianZimpelmann?

Reply:
I think for comprehensibility reasons it would be indeed good to use the data from the paper as input and transform them automatically, not by hand.

What kind of transformations were necessary? If it is not too complicated we could also just mention the transformations in the notes.

Reply:
> @TeBackh Do you have the names of the author? Another open question was whether the conversion of the test cases is comprehensible enough, since I made them by hand, or whether we need a function, for example. What do you think @ChristianZimpelmann?

Seibold, A. (2021). Reference points for retirement behavior: Evidence from german pension discontinuities. American Economic Review, 111(4), 1126-65.